app/src/main/cpp/Cylinder.cpp

```diff
@@ -28,6 +28,9 @@ namespace crow {
 // Ratio between world size and cylinder surface size.
 // It should match the values defined in WindowWidget.
 // 800px is the default window size for a 4m world size.
+// TODO: For now, this value remains the same even if the default window size changes
+//       to ensure that three windows at the default size cover 180° of the cylinder
+//       (which is not the case if we increase this value).
```

Comment:
The fact that we don't have to change this makes me thing that there is something that we don't really control. Perhaps the native code is not respecting this value somewhere but it's weird that it does not change. Mind double checking again why?